### DISCUSSION TITLE
any known sec issues?

### DISCUSSION THREAD

**Discussion by vasanthi-d** [TanStack/table#3713] [Q&A] [answered]
Hi All,

I am new to react-table and so far looking at examples this library looks promising. I would like to use it in my project and wonder if

1. there are any known security vulnerabilities with this library?
2. also, are the elements generated by this library like table, thead, tr etc... accessible?

**Comment by tannerlinsley** [comment, marked as answer]
1. All is secure. Carry on. 
2. It’s a headless library, so markup and accessibility are in your full control

**Reply by vasanthi-d** [reply]
Thank you very much for your response @tannerlinsley.
